extensions/wikia/AdEngine/js/MonetizationServiceHelper.js

```diff
@@ -0,0 +1,144 @@
+define('ext.wikia.adEngine.monetizationsServiceHelper', [
+	'jquery',
+	'wikia.geo',
+	'wikia.scriptwriter',
+	'wikia.tracker',
+], function ($, geo, scriptWriter, tracker) {
+	'use strict';
+
+	var track = tracker.buildTrackingFunction({
+			trackingMethod: 'internal',
+			category: 'monetization-module',
+			geo: geo.getCountryCode()
+		});
+
+	/**
+	 * @desc Inject content to the slot
+	 * @param {string} slot
+	 * @param {string} content
+	 * @param {function} success
+	 */
+	function injectContent(slot, content, success) {
+		scriptWriter.injectHtml(slot, content, function () {
+			var $container = $('#' + slot).find('.monetization-module');
+
+			trackImpression($container);
+			trackClickEvent($container);
+
+			success();
+		});
+	}
+
+	/**
+	 * @desc Get maximum number of ads for the page
+	 * @returns {number}
+	 */
+	function getMaxAds() {
+		var pageHeight = $('#WikiaArticle').height();
+		if (pageHeight > 5000) {
+			return 3;
+		} else if (pageHeight > 1500) {
+			return 2;
+		}
+		return 1;
+	}
+
+	/**
+	 * @desc Get country code
+	 * @returns {string}
+	 */
+	function getCountryCode() {
+		var countryCodeROW = 'ROW',
+			countryCodes = ['AU', 'CA', 'DE', 'GB', 'HK', 'JP', 'MX', 'RU', 'TW', 'US'],
```

Comment:
How often this list will change? What do you think about moving this array to `wg*` variable in WikiFactory?